Allow transformations on intermediate bindings streams produced during query execution (#1422)
comunica/comunica: packages/actor-query-operation-wrap-stream/README.md

# Comunica Wrap Stream Query Operation Actor

[![npm version](https://badge.fury.io/js/%40comunica%2Factor-query-operation-wrap-stream.svg)](https://www.npmjs.com/package/@comunica/actor-query-operation-wrap-stream)

A comunica Wrap Stream Query Operation Actor. This actor calls the `bus-iterator-transform` for all iterators produced by `query-operation` actors.

> This actor should only be included if you require the functionality of `bus-iterator-transform` as it will likely slow down query execution.

Reviewer: ```suggestion
This actor should only be included if you require the functionality of `bus-iterator-transform` as it may slow down query execution.
```
